### DISCUSSION TITLE
Welcome to Startup-Landing Discussions!

### DISCUSSION THREAD

**Discussion by neamatmim** [Startup-Landing/Startup-Landing#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ToteRio** [comment]
Hello friends, I am writing to you because it is impossible to use some of the templates, they all show the same error. What can you be doing??

C:\Users\machine\agency-modern-next>npm run dev
npm ERR! path C:\Users\machine\agency-modern-next\package.json
npm ERR! code ENOENT
npm ERR! errno -4058
npm ERR! syscall open
npm ERR! enoent ENOENT: no such file or directory, open 'C:\Users\machine\agency-modern-next\package.json'
npm ERR! enoent ENOENT: no such file or directory, open 'C:\Users\machine\agency-modern-next\package.json'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\machine\AppData\Roaming\npm-cache\_logs\2021-09-29T02_03_25_547Z-debug.log
